docs/es/docs/contributing.md

```diff
@@ -0,0 +1,497 @@
+
+Puedes <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request" class="external-link" target="_blank">agregar comentarios con sugerencias de cambio</a> en un pull request existente.
+
+Consulta la documentación acerca de <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews" class="external-link" target="_blank"> cómo agregar una revisión de pull request</a> para aprobarlo o pedir cambios.
+
+///
+
+* Verifica en <a href="https://github.com/fastapi/fastapi/discussions/categories/translations" class="external-link" target="_blank">GitHub Discussion</a> para coordinar traducciones en tu idioma. Puedes suscribirte a él, y cuando haya un nuevo pull request para revisar, un comentario automático será añadido a la discución.
+
+* Si traduces páginas, agregar un único pull request por cada página traducida. Esto hará mucho mas fácil de revisar por otros.
+
+* Consulta los códigos de 2-letras para el lenguaje que quieres traducir, puedes usar la tabla<a href="https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes" class="external-link" target="_blank">Lista de códigos ISO 639-1</a>.
+
+#### Idioma existente
+
+Digamos que quieres traducir una página para un idioma que ya tiene traducciones para algunas páginas, como el Español.
+
+En el caso del Español, el código de 2-letras sería `es`. Así que, el directorio para las traducciones en Español están ubicadas en `docs/es/`.
+
+/// tip
+
+El idioma principal ("oficial") es el inglés, y se encuentra ubicado en `docs/en/`.
+
+///
+
+Ahora ejecuta el live serrver para los documentos en español:
+
+<div class="termy">
+
+```console
+// Usa el comando "live" y pásale el código de idioma como argumento en la CLI
+$ python ./scripts/docs.py live es
+
+<span style="color: green;">[INFO]</span> Serving on http://127.0.0.1:8008
+<span style="color: green;">[INFO]</span> Start watching changes
+<span style="color: green;">[INFO]</span> Start detecting changes
+```
+
+</div>
+
+/// tip
+
+De otra manera, puedes hacer los pasos de los scripts manualmente.
+
+Ve al directorio del idioma, para las traducciones al Español sería en `docs/es/`:
+
+```console
+$ cd docs/es/
+```
+Entonces ejecuta `mkdocs` en ese directorio:
+
+
+```console
+$ mkdocs serve --dev-addr 8008
+```
+
+///
+
+Ahora puedes ir a <a href="http://127.0.0.1:8008" class="external-link" target="_blank">http://127.0.0.1:8008</a> y ver tus cambios en vivo.
+
+Ahora puedes ver que cada lenguaje tiene todas las páginas. Pero algunas de las páginas no están traducidas y tienen un cuadro de información en la parte superior, advirtiendo de que hace falta la traducción.
+
+Digamos que quieres añadir una traducción para la sección [Features](features.md){.internal-link target=_blank}.
+
+* Copiar el archivo en:
+
+```
+docs/en/docs/features.md
+```
+
+* Pégalo exactamente en la misma ubicación pero en el lenguaje que quieres traducir, e.j.:
```

Comment:
```suggestion
* Pégalo exactamente en la misma ubicación pero en el lenguaje que quieres traducir, p e.j.:
```

Reply:
I understand what you are saying but it's more common to see contractions for `por ejemplo` like `ej`

Reply:
```suggestion
* Pégalo exactamente en la misma ubicación pero en el lenguaje que quieres traducir, ej.:
```
Got it, in any case, "ejemplo" becomes "ej."